src/main/java/frc/robot/subsystems/swerve/SwerveCommands.java

```diff
@@ -23,11 +22,12 @@
 
 import java.util.Set;
 import java.util.function.DoubleSupplier;
+import java.util.function.Function;
 import java.util.function.Supplier;
 
 public class SwerveCommands {
 
-    public static SwerveCommands buildCommands(Swerve swerve){
+    public static SwerveCommands buildCommands(Swerve swerve) {
```

Comment:
you are already in swerve commands, consider renaming. (maybe to .build(), .create() or something like that)